vfs/rc_test.go

```diff
@@ -97,7 +98,15 @@ func TestRcPollInterval(t *testing.T) {
 	}
 	out, err := call.Fn(context.Background(), nil)
 	require.NoError(t, err)
-	assert.Equal(t, rc.Params{}, out)
```

Comment:
My understanding of this test is that it's just checking the reported polling interval. That changes for a backend with `ChangeNotify` enabled, so I have updated the expected output to match.